### ISSUE TITLE
Sub-objects at depth 3+ missing in HTTP response description

### ISSUE THREAD

**Issue by Simran-B** [arangodb/docs#106]
Compare [get_api_collection_figures.md](https://github.com/arangodb/arangodb/blob/devel/Documentation/DocuBlocks/Rest/Collections/get_api_collection_figures.md)
to https://www.arangodb.com/docs/3.4/http/collection-getting.html#return-statistics-for-a-collection

The response format is deeply nested, e.g:

- figures
  - dead
    - count
    - size

`count` and `size` are absent in the docs, as well as other nested objects at level 3:

![image](https://user-images.githubusercontent.com/7819991/61395376-98609180-a8c5-11e9-8936-68d2b5275250.png)

The definition looks okay:

```
@RESTSTRUCT{dead,collection_figures,object,required,collection_figures_dead}
the items waiting to be swept away by the cleaner

@RESTSTRUCT{count,collection_figures_dead,integer,required,int64}
The number of dead documents. This includes document
versions that have been deleted or replaced by a newer version. Documents
deleted or replaced that are contained the write-ahead log only are not reported
in this figure.

@RESTSTRUCT{size,collection_figures_dead,integer,required,int64}
The total size in bytes used by all dead documents.

@RESTSTRUCT{deletion,collection_figures_dead,integer,required,int64}
The total number of deletion markers. Deletion markers
only contained in the write-ahead log are not reporting in this figure.
```

**Comment by dothebart** [comment]
Its visible in the swagger description => needs to be some problem in the documentation system

**Comment by Simran-B** [comment]
Added to Technical Debt section of the Documenation Onboarding page in Confluence (Learn space).

**Comment by Simran-B** [comment]
Fixed by arangodb/docs#954
